kod/object/active/holder/room.kod

```diff
@@ -1779,9 +1779,9 @@ messages:
             for i in plActive
             {
                each_obj = First(i);
-               if IsClass(each_obj,&Monster)
+               if IsClass(each_obj,&Towns)
```

Comment:
Having to make this change is a good example of the crappiness of IsClass.  What do you think about this alternative design:

- ActiveObject gets a UserEntered() which by default does nothing.
- Instead of checking IsClass here, you always call UserEntered() on each active object.
- In Towns, UserEntered() does the logic that is now in ReadyForWelcome(), and calls a new function X if the user hasn't entered too recently.
- Rename the existing UserEntered to X.  This is tedious, but I think an improvement, as currently the name is misleading.  It is not called whenever a user enters.